css/stylesheet.css

```diff
@@ -0,0 +1,6 @@
+
+@media screen and (min-width: 400px) {
```

Comment:
That not how it works, the `@media` code should always be in under the same `style.css` file